examples/plugins/tasks/attractors/attractors_application.py

```diff
@@ -4,7 +4,7 @@
 from traits.api import Bool, Instance, List, Property
 
 # Local imports.
-from attractors_preferences import (
+from attractors.attractors_preferences import (
```

Comment:
This really does assume where `python` is run from, that's okay if this really isn't supposed to be run in any other way. `from .attractors_preferences import ...` would be free of that assumption in this case.

Reply:
i was on the fence about using implicit imports. updating them.

Reply:
hmm. that did not work as expected. trying to run `python run.py` gives the error

```
Traceback (most recent call last):
  File "run.py", line 7, in <module>
    from .attractors_plugin import AttractorsPlugin
ModuleNotFoundError: No module named '__main__.attractors_plugin'; '__main__' is not a package
```

which makes sense because python does not know about the `attractors` package because it is not on path when i cd into the `attractors` directory.

Reply:
OK... I am not sure how this is going to work under etsdemo if the examples need to assume certain python paths. Probably a separate issue.